pkgs/development/libraries/tinyxml-2/default.nix

```diff
@@ -0,0 +1,15 @@
+{ stdenv, fetchurl, cmake }:
+let version = "3.0.0";
+in stdenv.mkDerivation rec {
+  name = "tinyxml-2-${version}";
+  src = fetchurl {
```

Comment:
`fetchFromGitHub` is a more convenient way to achieve what you're doing here. Note, however, that it uses the checksum of the unpacked archive, not the archive itself, this is because github tends to change autogenerated tarballs every now and then (using different compression levels, for example).